<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="brigtnessSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="contrastSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="gammaSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>otwórz zdjęcie</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>zdjecie</string>
    </property>
   </widget>
   <widget class="QLabel" name="histogramLabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>110</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>contrast</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>gamma</string>
    </property>
   </widget>
   <widget class="QLabel" name="histogramG">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="histogramB">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>380</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="histogramR">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>jasnosc</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionactionExit">
   <property name="text">
    <string>actionExit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionactionOtworz">
   <property name="text">
    <string>actionOtworz</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
